torch/csrc/inductor/aoti_package/model_package_loader.cpp

```diff
@@ -445,6 +445,7 @@ AOTIModelPackageLoader::AOTIModelPackageLoader(
         }
       }
     }
+    delete[] filename;
```

Comment:
This could still memory leak on exception right? Can't we just put it in a unique_ptr with a proper user defined destructor?

Reply:
Or better yet, why not just stack allocate the array? Why both heap allocating it all???

Reply:
By stack allocation, do you mean Variable Length Array? I don't think that's supported by MSVC?

I can dig more with the previous std::string approach.